### PR TITLE
Mobs with Jaunt type movement can't Strip anymore

### LINKED ISSUE #7173: Revenants can strip people while invisible
Issue reported from  Round ID: 39054 (BeeStation Sage - Roleplay)

Reporting client version: 514.1583

What title says, revs can strip people while invisible, which is really annoying and they shouldn't be able to do that.

### PR DESCRIPTION
**Pull request by Miliviu**
## About The Pull Request
fixes #7173
Pretty clear

## Why It's Good For The Game

An invisible antagonist capable of passing through walls should not have the ability to stripe, it's simple!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

I've tested it!
I didn't feel it was necessary to record it because it was simple enough,.
</details>

## Changelog
:cl:
tweak: Mobs with jaunt type movement can't strip anymore
/:cl:

### CODE REVIEW

File: code/datums/elements/strippable.dm [hunk 1]
```diff
@@ -47,6 +47,9 @@
 	if(over != user)
 		return
 
+	if(isrevenant(user))
+		return
```
Comment (PowerfulBacon) [inline code/datums/elements/strippable.dm]: This should just check if the mob uses a jaunt movement type (eminence, revenent, other jaunt moving things/ghosts like demons)

Comment (Miliviu) [reply code/datums/elements/strippable.dm]: will do

**Review by PowerfulBacon — APPROVED** [review]

**Comment by St0rmC4st3r** [comment]
Looks like a feature to me.

**Comment by Oscari-pog** [comment]
This is really fucking funny and way better than revenant just trying his best to grief.

Being able to strip is honestly very harmless unless you genuinely have 0 perception.

I'd call this a feature.

**Comment by Miliviu** [comment]
I've read all the feedback, and I think you all are right, removing it will not make the game better.

I will try to touch this in a fun way!
Soon TM

**Comment by PowerfulBacon** [comment]
This is for all jauntjng mobs not just revenant, if you want a strio ability on revenant it should be a specific coded ability rather than a bug

**Pull request merged**